src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,33 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_TIME_FORMATTER
+            = DateTimeFormatter.ofPattern("dd.MM.yyyy");
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate startDate = LocalDate.parse(dateFrom, DATE_TIME_FORMATTER);
+        LocalDate endDate = LocalDate.parse(dateTo, DATE_TIME_FORMATTER);
+
+        StringBuilder reportBuilder = new StringBuilder("Report for period ")
+                .append(dateFrom).append(" - ").append(dateTo).append("\n");
+
+        for (String name: names) {
+            int periodSalary = 0;
+            for (int i = 0; i < data.length; i++) {
```

Comment:
let's use for-each here as well

Reply:
Agree. I thought index will be needed in the inner loop but it is not.